design/operater-gateway-support-migration-design.md

```diff
@@ -0,0 +1,88 @@
+# Contour Operator Gateway API Support Migration
+
+Status: Draft
+
+## Abstract
+Currently the Contour Operator and Contour Controller have both started implementing Gateway API support.
+Both components contain controllers that reconcile GatewayClass and Gateway resources and that conflict when reconciling status.
+This document describes options for how we should proceed to untangle these.
```

Comment:
Thanks Sunjay.  Since there are so many moving parts to Contour + Gateway API + Operator, I'm going to try to be more precise with the language and ask for some clarifications